node/src/types/timestamp.rs

```diff
@@ -300,7 +300,6 @@ impl FromBytes for TimeDiff {
     }
 }
 
-#[cfg(test)]
```

Comment:
Does this need to be removed?

Reply:
The `From<Duration> for TimeDiff` impl isn't exposed to the test, and I think this is a worthwhile implementation to have. Is there a reason why it should stay as `#[cfg(test)]`?

Reply:
I'm guessing it's to keep the API as small as possible for the production case, but if we need it then fair enough.  I don't see a problem exposing it - I just wanted to check we did need the change.

Reply:
Yeah, I'm using a Duration to construct a TimeDiff, and I think it's a worthwhile implementation to have for interop.